docs/src/main/asciidoc/infinispan-client.adoc

```diff
@@ -42,15 +40,67 @@ This will add the following to your pom.xml
 </dependency>
 ----
 
-The Infinispan client is configurable in the `application.properties` file that can be
-provided in the `src/main/resources` directory. These are the properties that
-can be configured in this file:
+== Configuring the Infinispan client
 
-include::{generated-dir}/config/quarkus-infinispan-client.adoc[opts=optional, leveloffset=+1]
+Open the `application.properties` file in the `src/main/resources` directory with any text editor.
+
+Note that Infinispan documentation refers to a `hotrod-client.properties` file.
+You can configure the Infinispan client with either properties file but `hotrod-client.properties` always takes
+priority over `application.properties`.
+
+Additionally, you cannot update configuration properties at runtime.
+If you modify `application.properties` or `hotrod-client.properties`, you must rebuild the application before those changes take effect.
+
+== Connecting to Infinispan clusters
+
+Add the following properties to connect to Infinispan Server:
+
+[source,properties]
```

Comment:
Hi @karesti , I'm checking the updates and trying to follow the steps however I got into some trouble :) 

- When I used a sample hello project with this configuration I was able to  open the console at http://127.0.1.1:11222/console/ and follow the https://infinispan.org/get-started/. There was just one little confusion between 127.0.1.1:11222 and localhost:11222. localhost:11222 didn't work.

- Then I tried the same with the quickstart and I wasn't able to compile the quickstart and failed to connect to localhost/127.0.0.1:11222. I'm probably missing some information. 
- I noticed that the configuration in the quickstart is different from the one in this guide. Which one should I use?

I have limited knowledge about Quarkus and this is my first experience with Infinispan however I wanted to share this with you in case that some of it is relevant. Please let me know if you want me to share the errors I got with the infinispan-client-quickstart. I'm on zulip or gchat.

Reply:
The configuration in the quickstart and the guide are the same now, I did a PR to remove the unnecessary default values and simplify the connection.
 `http://localhost:11222/console/welcome` and `http://127.0.0.1:11222/console/welcome` are the same url that point to the local running infinispan server (with a container or downloading the server.
maybe you are missing a config in your `hosts` file?
thanks for trying the quickstart!